logging/src/detailed_json.rs

```diff
@@ -60,5 +62,12 @@ pub fn default<W>(io: W) -> slog_json::Json<W>
 where
     W: io::Write,
 {
-    new_with_ts_fn(io, |_: &Record| chrono::Local::now().to_rfc3339()).build()
+    const INVALID_TIME: &str = "invalid timestamp";
+    new_with_ts_fn(io, |_: &Record| match OffsetDateTime::now_local() {
```

Comment:
Guys, here local timestamp is used. Wouldn't it be better to use UTC instead?

Reply:
As I said above, I agree using UTC everywhere, as the block timestamps are UTCs, and when we are troubleshooting stuff the blocks and their timestamps are referenced a lot.